packages/velo-external-db-commons/lib/schema_commons.js

```diff
@@ -113,5 +109,12 @@ const supportedSchemaOperationsFor = (impl) => {
     }
 }
 
+const allowedOperationsFor = (collection) => {
+    return collection.fields.find(c => c.field === '_id') ? ReadWriteOperations : ReadOnlyOperations 
+}
+
+const prepareFieldsList = (fields) => fields.map(f => ({ ...f, queryOperators: QueryOperatorsFor[f.type] }))
```

Comment:
change the function name to something more meaningful, 
and i don't like QueryOperatorsFor as well, it's nice but it's a constant array and not a function